arrow/api.py

```diff
@@ -51,5 +48,4 @@ def factory(type):
     return ArrowFactory(type)
 
 
-__all__ = ['get', 'utcnow', 'now', 'factory', 'iso']
-
+__all__ = ['get', 'utcnow', 'now', 'factory']
```

Comment:
`iso` doesn't exist any more, and was causing a warning when generating the docs.